doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -634,6 +634,88 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notification messages
+
+Notifications include two fields:
+
+* `subscriptionId` which represents the concerned subscription that originates the notification
+* `data` an array with the notification data itself which includes the entity and attributes concerned. Each
+  element in the array corresponds to a different entity. The way of representing each entity depends on
+  the `attrsAs` parameter associated to the subscription, as shown below.
+
+
+If `attrsAs` is `normalized` then default entity representation is used:
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [
+     {
+       "id": "Room1",
+       "type": "Room",
+       "temperature": {
+         "value": 23,
+         "type": null,
+         "metadata": {}
+       },
+       "humidity": {
+         "value": 70,
+         "type": "percentage",
+         "metadata": {}
+       }
+     },
+     {
+     "id": "Room2",
+     "type": "Room",
+     "temperature": {
+       "value": 24,
+       "type": null,
+       "metadata": {}
+     }
+   ]
+}
+```
+
+If `attrsAs` is `kevValues` then keyValues entity partial representation mode is used:
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [
+     {
+       "id": "Room1",
+       "type": "Room",
+       "temperature": 23,
+       "humidity": 70
+       }
+     },
+     {
+     "id": "Room2",
+     "type": "Room",
+     "temperature": 24
+     }
+   ]
+}
+```
+
+
+If `attrsAs` is `values` then values entity partial representation mode is used:
+
+```
+{
+   "subscriptionId": "12345",
+   "data": [ [23, 70], [24] ]
+}
+```
+
+Notifications must include a `X-Ngsiv2-AttrsAs` header with the value of `attrsAs` of the associated subscription,
+so notification receiver can know the format used without needing processing notification payload.
+
+### Notification templates
+
+Based on the "template" field defined as part of the subscription and in a macro replacement language
+that could be used in "template" "callback", "headers" and "query". Details TBD.
+
```

Comment:
We already have some notes of that kind along the .apib. Thus, one more doesn't hurt (and can be useful to highly where the spec needs to be completed).

NTC